server/models.py

```diff
@@ -23,6 +31,25 @@ def full_name(self):
             return self.last_name
         return "%s %s" % (self.first_name, self.last_name)
 
+    @property
+    def is_authenticated(self):
+        return True
+
+    @property
+    def is_active(self):
+        return self.active
+
+    @property
+    def is_admin(self):
+        return self.admin
+
+    @property
+    def is_anonymous(self):
+        return False
+
+    def get_id(self):
+        return str(self.id)
```

Comment:
They are the part of [Flask-Login] (https://flask-login.readthedocs.org/en/latest/#your-user-class) specification. We will have anonimous user if it wasn't logged in. In future, I think,  we will be able replace our `client-id` with it.